druntime/src/rt/dwarfeh.d

```diff
@@ -13,7 +13,7 @@ module rt.dwarfeh;
 
 // debug = EH_personality;
 
-version (Posix):
```

Comment:
Did you not consider Cygwin?

Reply:
What do you mean? Cygwin builds will be broken?

Reply:
Even Mingw is a dwarf eh platform. OK, dmd has no intention of supporting either, but doesn't ldc also use this module?

Reply:
The point I'm raising is that it is not without good reason that `version(!Windows)` is not supported by the language. It is very bad practice to work around it by using `version(Windows){} else:` instead.

That said, this module isn't used by gdc, so I'll defer it to an ldc developer to check this.

Reply:
> The point I'm raising is that it is not without good reason that version(!Windows) is not supported by the language. It is very bad practice to work around it by using version(Windows){} else: instead.

Can we define enum/version like `enableDwarfEH` and then set if for needed targets?

Reply:
> The point I'm raising is that it is not without good reason that version(!Windows) is not supported by the language. It is very bad practice to work around it by using version(Windows){} else: instead.

What do you think about ~same https://github.com/dlang/dmd/pull/16141 ?